roles/ceph-common/templates/restart_mons_daemons.sh.j2

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+RETRIES="{{ handler_health_mon_check_retries }}"
+DELAY="{{ handler_health_mon_check_delay }}"
+MONITOR_NAME="{{ monitor_name }}"
+CLUSTER="{{ cluster }}"
+
+systemctl restart ceph-mon@${MONITOR_NAME}
+# We need to wait because it may take some time for the socket to actually exists
+sleep 2
```

Comment:
this is really prone to error, please do a loop and wait for 10 secs, then exit 1 with a message.